blob/gcsblob/gcsblob.go

```diff
@@ -43,7 +43,7 @@
 //  - Reader: *storage.Reader
 //  - Attributes: storage.ObjectAttrs
 //  - CopyOptions.BeforeCopy: *storage.Copier
-//  - WriterOptions.BeforeWrite: *storage.Writer
+//  - WriterOptions.BeforeWrite: **storage.ObjectHandle, *storage.Writer
```

Comment:
The double (and thus triple as seen below) pointer gives me pause. I could imagine providing some larger API surface to smooth it out, but it's likely not worth it.

Reply:
Agree.